packages/polling/src/poll.ts

```diff
@@ -153,6 +153,17 @@ export class Poll<T = any, U = any, V extends string = 'standby'>
     return this._ticked;
   }
 
+  /**
+   * Return an async iterator that yields every tick.
+   */
+  async *[Symbol.asyncIterator](): AsyncIterableIterator<IPoll.State<T, U, V>> {
+    yield this.state;
+    while (!this.isDisposed) {
+      await this.tick.catch(() => undefined);
+      yield this.state;
+    }
```

Comment:
Should we do it the other way around to always check for disposable state:

```suggestion
    while (!this.isDisposed) {
      yield this.state;
      await this.tick.catch(() => undefined);
    }
```

Reply:
This suggestion will work, but it will never `yield` the `disposed` state and one of the tests I wrote will fail.

There is a real question of whether we *want* to `yield` the disposed state, but that was my motivation for writing it this way.

Reply:
Hm, we do *not* emit a `ticked` signal when the poll is disposed. And we do not `resolve` the `tick` promise, we `reject` it. So maybe we should not `yield` when disposed either.

Reply:
I think I talked myself into it.